src/api/v1alpha1/inspectionpolicy_types.go

```diff
@@ -114,6 +114,24 @@ type Assessment struct {
 	// ElasticSearch certificate for the client
 	// +kubebuilder:validation:Optional
 	OpenSearchCert string `json:"openSearchCert"`
+	// Indicate whether to config of governor
+	Governor Governor `json:"governor"`
+}
+
+// Governor contains policies for governor to send report
+type Governor struct {
+	// Indicate whether to send the reports to governor
+	// +kubebuilder:default:=false
+	Enabled bool `json:"enabled"`
+	// Unique identifier of the cluster
+	// +kubebuilder:validation:Optional
+	ClusterID string `json:"clusterId"`
+	// Api url to send telemetry data
+	// +kubebuilder:validation:Optional
+	URL string `json:"url"`
```

Comment:
In the case Enabled=true, are we validating Other fields are not empty?

If we can't do conditional validation base on Enabled using KubeBuilder. Then at least we need handle these checks at code level. This can cause crashloopback while running